source/stats-page/stats.html

```diff
@@ -16,92 +17,86 @@
     <script src="/components/stats-card/stats-card.js" defer="defer"></script>
     <script src="/stats-page/stats.js" defer="defer"></script>
     <style>
-      button {
-        color: var(--btn-color);
-        background: var(--linear-gradient-dark);
-        box-shadow: 2px 2px 3px var(--medium-color);
-        cursor: pointer;
-
-        margin: 5px;
-        padding: 10px;
-        border: none;
-        font-size: 2rem;
-        text-align: center;
-        font-weight: lighter;
-        border-radius: 1rem;
-      }
-
-      button:hover {
-        animation: button-pop;
-        transform: scale(1          .2);
-        animation-duration: 0.25s;
-      }
-
-      button:active {
-        animation: button-click;
-        transform: scale(1);
-        animation-duration: 0.25s;
-      }
-      
-      h1 {
-        color: var(--primary-color);
-        font-size: 2.5rem;
-        margin: 0;
-      }
-
-      
-      main {
-        display: grid;
-        padding-bottom: 5em;
-        margin: 1rem;
-      }
-
-      #heading-container {
-        align-items: center;
-        border-bottom: 2px solid var(--medium-light-color);
-        display: flex;
-        flex-wrap: wrap;
-        justify-content: space-between;
-        margin: 0 1rem 2% 1rem;
-        padding-bottom: 0.5rem;
-      }
-
-      #heading-container-btns {
-        display: flex;
-      }
-      
-      @keyframes button-pop {
-        0% {
-          transform: scale(1);
+        button {
+            color: var(--btn-color);
+            background: var(--linear-gradient-dark);
+            box-shadow: 2px 2px 3px var(--medium-color);
+            cursor: pointer;
+            margin: 5px;
+            padding: 10px;
+            border: none;
+            font-size: 2rem;
+            text-align: center;
+            font-weight: lighter;
+            border-radius: 1rem;
         }
         
-        50% {
-          transform: scale(1.4);
+        button:hover {
```

Comment:
Can you clarify which of these changes are the result of your linter and which are intentional? We should avoid making any unnecessary changes to the HTML right now since our code and tests are extremely brittle and rely on the tags.

Reply:
Oh shoot good catch. I am gonna redo the pull request.